README.md

```diff
@@ -75,7 +90,7 @@ device by putting the device into pairing mode and turning on a device id, light
 
 The device ids don't have to be unique, but it's advisable.
 
-Lighting2
+HomeEasy (EU)
```

Comment:
I've pushed a fix to the typo, and clarified the description of the lighting nodes. Also updated the description of the disconnect/reconnect mechanism now it doesn't throw an error.